pkg/api/plugin.go

```diff
@@ -11,8 +11,9 @@ import (
 type ContextKey string
 
 const (
-	ContextKeyClientAuthorizer ContextKey = "ClientAuthorizer"
-	ContextAcceptLanguages     ContextKey = "AcceptLanguages"
+	ContextKeyClientAuthorizer      ContextKey = "ClientAuthorizer"
+	ContextKeyVaultClientAuthorizer ContextKey = "VaultClientAuthorizer"
```

Comment:
@asalkeld need to highlight this new key and how to use it to MSFT in the release notes